drawing/tools/custom/Vector.js

```diff
@@ -352,7 +351,11 @@ if(0 && dojox.drawing.defaults.zAxisEnabled){
 					}
 				},this);
 			};
-			dojo.connect(this, "onRenderStencil", this, function(){ if(this.zSelected){ this.zDeselect(this.zSelected)}});
+			dojo.connect(this, "onRenderStencil", this, function(){ 
```

Comment:
Given the other changes, it would be nice to replace connect statements with dojo/on and/or aspect.after, but I realize that's non-trivial.